tests/testing_touch.cpp

```diff
@@ -8,19 +8,59 @@ int main()
 {
   Touch touch;
 
-  int touchSize = 16;
-  int discrete_touch[16] = {0};
+  const int touchSize = 16;
+  int discrete_touch[touchSize] = {0};
 
-  // simulate a touch (1) at a position 5
+  // simulate a blob of size 1 starting at position 0
```

Comment:
ideally for the tests I think it would be good to move to a proper test framework, such as Catch2.

https://github.com/catchorg/Catch2

Reply:
sure -- probably belongs in its own PR though?

Reply:
I agree :):